lib/auth/oidc.go

```diff
@@ -835,32 +835,42 @@ func (a *Server) getClaims(oidcClient *oidc.Client, connector services.OIDCConne
 			return nil, trace.Wrap(err)
 		}
 
-		serviceAccountURI := connector.GetGoogleServiceAccountURI()
-		if serviceAccountURI == "" {
-			return nil, trace.NotFound(
-				"the gsuite connector requires google_service_account_uri parameter to be specified and pointing to a valid google service account file with credentials, read this article for more details https://developers.google.com/admin-sdk/directory/v1/guides/delegation")
+		var config *jwt.Config
+		var jsonCredentials []byte
```

Comment:
Add a variable here which stores a string representation of the field used to provide the service account and then output it in any error message related to credential loading - it'll make debugging easier.

```suggestion
		var jsonCredentials []byte
		var credentialLoadingMethod string
```